client/blocks/author-selector/docs/example.jsx

```diff
@@ -15,7 +15,12 @@ import { getCurrentUser } from 'calypso/state/current-user/selectors';
 function AuthorSelectorExample( { primarySiteId, displayName } ) {
 	return (
 		<Card>
-			<AuthorSelector siteId={ primarySiteId } allowSingleUser popoverPosition="bottom">
+			<AuthorSelector
+				key={ primarySiteId }
```

Comment:
Disregarding that the `key` prop is now in the example of this component, would it be an improvement if we explain the need for it in the README? Otherwise, if anyone else intends to consume this component, they might miss adding a `key` and fall into the same trap again.

Reply:
I tend to agree with Jarda on this one. Using `key` is a pretty solid and straightforward pattern for that purpsoe which only has one downside: it's not always obvious. I've found that adding an inline comment to illustrate the purpose of the `key` prop being present has helped with that.